docs/source/conf.py

```diff
@@ -37,7 +37,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'PSG Services Documentation'
-copyright = '2021, IOHK'
```

Comment:
You don't need to update copyright, it's from the year you claim the copyright... so leave it at 2021.

Reply:
Fixed.